src/db/models/endpoints.js

```diff
@@ -28,7 +28,7 @@ const endpointSchema = new mongoose.Schema(
       }
     },
     transformation: {
-      input: {type: String, enum: ALLOWED_CONTENT_TYPES, required: true},
+      input: {type: String, enum: ALLOWED_CONTENT_TYPES},
```

Comment:
Lets try and change this to be required if the endpoint method contains a POST/PUT request. We can conditional make a field required based on another value
https://github.com/Automattic/mongoose/issues/5119

This will be important if the user does a POST request with JSON but we dont what to validate the Content_type on to conforming the parsing

Reply:
fixed in https://github.com/jembi/openhim-mediator-mapping/pull/79/commits/3a00fb441b2e343f908c15bfdda0f49587f6b4f7